File: benchmark/2026-09-17-7677581(latest)/semantic-compression/probes/java/F11.P2.java
import java.util.ArrayList;
import java.util.List;

class Main {
    static int subRange() {
        var xs = new ArrayList<>(List.of(10, 20, 30, 40, 50));
        // BEGIN PROBE F11.P2
        var part = xs.subList(1, 4);
        return part.get(0);
        // END PROBE F11.P2
    }

    public static void main(String[] args) {
        System.out.println(subRange());
    }
}

File: benchmark/2026-09-17-7677581(latest)/semantic-compression/probes/cpp/F11.P2.cpp
#include <cstdio>
#include <span>
#include <vector>

int second(std::vector<int>& xs) {
// BEGIN PROBE F11.P2
    std::span part{xs.begin() + 1, 3};
    return part[0];
// END PROBE F11.P2
}

int main() {
    std::vector<int> xs{10, 20, 30, 40, 50};
    std::printf("%d\n", second(xs));
}
